src/lib/viewers/doc/DocBaseViewer.js

```diff
@@ -163,13 +163,19 @@ class DocBaseViewer extends BaseViewer {
         this.startPageNum = this.getStartPage(this.startAt);
 
         if (this.options.enableThumbnailsSidebar) {
-            this.thumbnailsSidebarEl = document.createElement('div');
+            this.thumbnailsSidebarEl = document.createElement('nav');
             this.thumbnailsSidebarEl.className = `${CLASS_BOX_PREVIEW_THUMBNAILS_CONTAINER}`;
             this.thumbnailsSidebarEl.setAttribute('data-testid', 'thumbnails-sidebar');
+            this.thumbnailsSidebarEl.setAttribute('aria-label', __('thumbnail_label'));
             this.thumbnailsSidebarEl.tabIndex = 0;
             this.rootEl.insertBefore(this.thumbnailsSidebarEl, this.containerEl);
         }
 
+        if (this.containerEl) {
+            this.containerEl.setAttribute('aria-label', __('document_preview_label'));
```

Comment:
Should this only be done for document previews? Should all file previews set the `aria-label` and `role` on `this.containerEl`?

Reply:
good question, yes we can set those attributes for all-files but changing the message from 'document preview' to something else ex. "file preview"

Reply:
makes sense?

Reply:
Yes, the type of file being previewed is known by the specific viewers. Does this label get read by the screen reader? 

Could we possibly put these `role` and `aria-label` directly in the [shell.html](https://github.com/box/box-content-preview/blob/master/src/lib/shell.html#L62) instead of adding it programmatically?

Reply:
I agree it would be preferable to keep the message generic, such as "file preview," and put it in `shell.html` as suggested above.

Reply:
@ConradJChan, it looks like the `aria-label` value needs to be injected via JS, since it's a translated string.